var/spack/repos/builtin/packages/py-functools32/package.py

```diff
@@ -0,0 +1,41 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the LICENSE file for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class PyFunctools32(Package):
+    """Backport of the functools module from Python 3.2.3 for use on 2.7 and
+    PyPy."""
+
+    homepage = "https://github.com/MiCHiLU/python-functools32"
+    # base https://pypi.python.org/pypi/functools32
+    url      = "https://pypi.python.org/packages/source/f/functools32/functools32-3.2.3-2.tar.gz"
+
+    version('3.2.3-2', '09f24ffd9af9f6cd0f63cb9f4e23d4b2')
+
+    extends('python', type=nolink)
```

Comment:
Does this actually work? I've never seen something extend Python with a type of nolink before. Either way, it probably isn't necessary.